configs/mainnet.yaml

```
@@ -165,6 +165,8 @@ DATA_COLUMN_SIDECAR_SUBNET_COUNT: 128
 MAX_REQUEST_DATA_COLUMN_SIDECARS: 16384
 SAMPLES_PER_SLOT: 8
 CUSTODY_REQUIREMENT: 4
+VALIDATOR_CUSTODY_REQUIREMENT: 8
+BALANCE_PER_ADDITIONAL_CUSTODY_SUBNET: 32000000000 # 2**5 * 10**9 (= 32,000,000,000)
```

This should probably be custody group since we've [decoupled the DAS subnets](https://github.com/ethereum/consensus-specs/pull/3832)

```suggestion
BALANCE_PER_ADDITIONAL_CUSTODY_GROUP: 32000000000 # 2**5 * 10**9 (= 32,000,000,000)
```